<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Converter</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="zoominButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset resource="ImageConverter.qrc">
        <normaloff>:/main/zoomin.png</normaloff>:/main/zoomin.png</iconset>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>0</number>
      </property>
      <property name="autoRepeatInterval">
       <number>33</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="zoomSlider">
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="zoomoutButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset resource="ImageConverter.qrc">
        <normaloff>:/main/zoomout.png</normaloff>:/main/zoomout.png</iconset>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>0</number>
      </property>
      <property name="autoRepeatInterval">
       <number>33</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="locale">
       <locale language="Russian" country="RussianFederation"/>
      </property>
      <property name="title">
       <string>Изменение пропорций изображения</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="Proportion">
         <property name="locale">
          <locale language="Russian" country="RussianFederation"/>
         </property>
         <property name="text">
          <string>Сохранять пропорции</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="Height">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="locale">
          <locale language="Russian" country="RussianFederation"/>
         </property>
         <property name="text">
          <string>Высота</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="Width">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="locale">
          <locale language="Russian" country="RussianFederation"/>
         </property>
         <property name="text">
          <string>Ширина</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="locale">
       <locale language="Russian" country="RussianFederation"/>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="RussianFederation"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="Russian" country="RussianFederation"/>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="Russian" country="RussianFederation"/>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout_Program"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="actionAbout_Program">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>О Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageConverter.qrc"/>
 </resources>
 <connections/>
</ui>
